store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9496,5 +9496,10 @@ TODO: delete them permanently from here
   <globalConfigValue>TRUE</globalConfigValue>
   <desc>Whether to enable zimbra network next generation modules.</desc>
 </attr>
+
+<attr id="2118" name="zimbraSieveRequireControlRFCCompliant" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited" since="8.7.8">
```

Comment:
I think we should make the attribute TRUE by default otherwise it will never get used.